geoh5py/data/data.py

```diff
@@ -119,6 +117,38 @@ def copy(
 
         return new_entity
 
+    def copy_from_extent(
```

Comment:
Can we add a unitest for this? You can probably just modify the copy from extent test on the object, and check that the call on the data yields the same outcome.